tests/e2e/default/enketo/death-report.wdio-spec.js

```diff
@@ -0,0 +1,78 @@
+const moment = require('moment');
+const utils = require('../../../utils');
+const commonPage = require('../../../page-objects/default/common/common.wdio.page');
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const userFactory = require('../../../factories/cht/users/users');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const personFactory = require('../../../factories/cht/contacts/person');
+const contactPage = require('../../../page-objects/default/contacts/contacts.wdio.page');
+const reportsPage = require('../../../page-objects/default/reports/reports.wdio.page');
+const analyticsPage = require('../../../page-objects/default/analytics/analytics.wdio.page');
+const genericForm = require('../../../page-objects/default/enketo/generic-form.wdio.page');
+const deathReportForm = require('../../../page-objects/default/enketo/death-report.page');
+
+describe('Submit a death report', () => {
+  const places = placeFactory.generateHierarchy();
+  const healthCenter = places.find(place => place.type === 'health_center');
+  const offlineUser = userFactory.build({ place: healthCenter._id, roles: ['chw'] });
+  const person = personFactory.build({ parent: {_id: healthCenter._id, parent: healthCenter.parent} });
+
+  before(async () => {
+    await utils.saveDocs([...places, person]);
+    await utils.createUsers([offlineUser]);
+    await loginPage.login(offlineUser);
+  });
+
+  it('Should submit a death report', async () => {
+    const deathDate = moment().subtract(1, 'day');
```

Comment:
if the computer current date is end of the month or first day of the month, will the target still count correctly? Because I think targets consider records in the current month 🤔
But I don't remember if this target is configured to use the report's submit date or the death date.

Reply:
You are right again, and today was the perfect day to test that, I didn't have to modify anything, and it failed, so I changed it to the current day.
Do you think that it could be another scenario where it can failed?

Reply:
No, at the moment 🤔. The safest is to set a date in the middle of the month